apis/python/src/tiledbsoma/soma_sparse_nd_array.py

```diff
@@ -266,9 +264,9 @@ def read_as_pandas_all(
         row_ids: Optional[Sequence[int]] = None,
```

Comment:
Just a heads up: this hard-wired 2D assumption is a bug (fixed in `main`).  Need to make sure this is merged correctly with main. 

I'd like to re-review post-merge.

Reply:
@bkmartinjr i indeed have recently merged PRs to rebase onto! :)